### ISSUE TITLE
Design Oversight? - Min and Max inputs on slider nodes.

### ISSUE THREAD

**Issue by eric-schleicher** [vizorvr/vizor#1641]
It would be nice to be able to arbitrarily calculate mins and max values for sliders in the graph rather than only have direct entry.

**Comment by gmarinov** [comment]
![screen shot 2016-05-04 at 21 50 04](https://cloud.githubusercontent.com/assets/14101296/15028665/dc877fcc-1241-11e6-9650-b337302400f0.png)

http://vizor.io/vbYj

**Comment by eric-schleicher** [comment]
I was thinking on the actual manipulator.  More like this.
![image](https://cloud.githubusercontent.com/assets/8261783/15032997/71136a42-121b-11e6-9664-56b057760aa6.png)

**Comment by gmarinov** [comment]
yeah, considering this can be done with a subgraph and three terminals, it becomes a design issue at the heart of visual programming: should we be building macros? it's still an open question.

**Comment by kschzt** [comment]
@gmarinov what is a terminal? do you mean input? the issue at heart here is that there is no way to programmatically set the min/max of the slider itself, which is a small design oversight

**Comment by gmarinov** [comment]
@kschzt I used 'terminal' for how a proxy looks from the outside of the subgraph node (input or output). The issue would be one or another depending on the use case 

 - a slider with inputs that sometimes requires extra nodes to define min and max programmatically, 
 - or a slider that always requires extra nodes to define the boundaries. 

It's not strictly a design oversight. It can be done with a very simple flow, and it can be saved as a macro.

<img width="456" alt="screen shot 2016-05-09 at 12 12 31" src="https://cloud.githubusercontent.com/assets/14101296/15111502/5873a0c0-15df-11e6-9baf-eefeb7cf9368.png">
